notebooks/llm-chatbot/llm-chatbot.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #98.            "qwen-7b-chat": {"sym": True, "group_size": 128, "ratio": 0.6},
</pre><p>it should be reformatted to align with other config</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1901/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>